src/api/comments.ts

```diff
@@ -0,0 +1,27 @@
+import { BASE_URL } from './api';
+
+export const getPostComments = async (postId: number) => {
+  const response = await fetch(`${BASE_URL}/comments`);
+  const userPosts = await response.json();
+
+  return userPosts.filter((comment: Comment) => comment.postId === postId);
+};
```

Comment:
use this ability of API to load needed comments 
![image](https://user-images.githubusercontent.com/64437769/134147622-e39f844d-56a9-43cc-a45e-4726a64be985.png)